src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/TagParser.java

```diff
@@ -214,9 +214,27 @@ private static boolean isCommentTag(String[] text, Point pos) {
      */
     private static Point skipHtmlComment(String[] text, Point fromPoint) {
         Point toPoint = fromPoint;
+
+        // Find next occurrence of '>'
+        String line = text[toPoint.getLineNo()];
+        int charIndex = line.indexOf('>', toPoint.getColumnNo());
+        if (charIndex != -1) {
+            toPoint = new Point(toPoint.getLineNo(), charIndex);
+        }
+
         while (toPoint.getLineNo() < text.length && !text[toPoint.getLineNo()]
                 .substring(0, toPoint.getColumnNo() + 1).endsWith("-->")) {
-            toPoint = findChar(text, '>', getNextPoint(text, toPoint));
```

Comment:
Why not:` toPoint = getNextPoint(text, toPoint);`? Isn't the mutation?

Reply:
I did this only initially but that resulted in pitest-javadoc build failure which mentioned addition of yet another extra suppression. Considering that as not a good thing, a new change was done this way which in the latest PR.

Reply:
For the new change, pitest-javadoc is passing but we have another failures now :(

Reply:
Looks like new code is causing some infinite loop or stackoverflow. Please share new surviving mutation after https://github.com/checkstyle/checkstyle/pull/14181/files#r1438614815, this may be key in finding a test case.

Reply:
Okay so now we don't have a pitest-javadoc failure, looks like a new merge for the method https://github.com/checkstyle/checkstyle/pull/14185/files did the magic

Reply:
this mutation surviving because it is micro performance optimization:
 instead of position by positton search for `-->` we did search for `>` as we know that only this symbols is good marked of some tag closure.